CleverPush/Source/CPAppBannerCarouselBlock.h

```diff
@@ -18,6 +18,7 @@
 
 @property (nonatomic, strong) NSString *id;
 @property (nonatomic, strong) NSMutableArray<CPAppBannerBlock*> *blocks;
+@property (assign) BOOL isScreenClicked;
```

Comment:
Why do we need this - can't we find an easier solution?

Reply:
@gbhrdt The above flag is handled for multiple screens because it would be used to detect that this screen had already been used if there were multiple screens.